sympy/sets/tests/test_sets.py

```diff
@@ -1046,6 +1046,16 @@ def test_issue_10931():
     assert S.Integers - S.Reals == EmptySet()
 
 
+def test_issue_11442():
+    a = Symbol('a', real=True)
+    b = Symbol('b', real=True)
+    i1 = Interval(2, 7)
+    i2 = Interval(a, b)
+    assert not i1.is_EmptySet
```

Comment:
Did you check with `S.false`(SymPy bool) or `False` (Python bool)?


On 08/06/2016 11:13 PM, Ruslan Pisarev wrote:
>
> In sympy/sets/tests/test_sets.py 
> <https://github.com/sympy/sympy/pull/11483#discussion_r73789357>:
>
> > @@ -1046,6 +1046,16 @@ def test_issue_10931():
> >      assert S.Integers - S.Reals == EmptySet()
> >
> >
> > +def test_issue_11442():
> > +    a = Symbol('a', real=True)
> > +    b = Symbol('b', real=True)
> > +    i1 = Interval(2, 7)
> > +    i2 = Interval(a, b)
> > +    assert not i1.is_EmptySet
>
> I know it, but it failed. i1.is_EmptySet gives us False, but that 
> False doesn't equal False... It was strange for me and I changed test 
> for not
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/sympy/sympy/pull/11483/files/42cf353ac76ef078df3b155cde32f511a8ff5f58#r73789357>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AHWt8QEE6SPTE7oWOTyUAeCUqOO-55Orks5qdMfAgaJpZM4JeUkO>.
>